pkg/util/helm_helpers.go

```diff
@@ -299,6 +259,30 @@ func GetWithHelm3(releaseName, namespace, kubeConfig string) (*release.Release,
 	return nil, fmt.Errorf("unable to find release '%s' in namespace '%s'", releaseName, namespace)
 }
 
+// GetBlackDuckChartAndReleaseWithHelm3 checks whether an file exist in the chart. If exist, it will convert the file to map and return the map
+func GetBlackDuckChartAndReleaseWithHelm3(namespace, kubeConfig, chartURL, fileName string) (map[string]interface{}, error) {
+	actionConfig, err := CreateHelmActionConfiguration(kubeConfig, "", namespace)
+	if err != nil {
+		return nil, err
+	}
+
+	chart, err := LoadChart(chartURL, actionConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load release at '%s' for getting size: %s", chartURL, err)
+	}
+	validInstallableChart, err := isChartInstallable(chart)
+	if !validInstallableChart {
+		return nil, err
+	}
+
+	fileValues := map[string]interface{}{}
+	if err := mergeExtraFilesToConfig(chart, fileValues, []string{fileName}); err != nil {
```

Comment:
Can we rename this function to something more clear - like mergeWithExtraFilesFromChart() ?
* It isn't doing anything with the config